lib/tasks/test.js

```diff
@@ -67,7 +67,15 @@ module.exports = function(gulp, config) {
 		])
 		.then(function() {
 			if (config.browserTest) {
-				return module.exports.browserTest(gulp, config);
+				return module.exports.browserTest(gulp, config).then(function(){
```

Comment:
Alright, then could you add a space after the function keyword please?

And re tests, I probably should of just disabled them instead of deleting them, sorry about that